docs/_includes/footer.html

```diff
@@ -1,9 +1,22 @@
 <div class="usa-grid">
-  <div class="usa-width-one-half">
-    <p>Feedback? Create an issue on the <a href="{{ site.repos[0].url }}">GitHub repository</a> or email us at <a href="mailto:uswebdesignstandards@gsa.gov">uswebdesignstandards@gsa.gov</a>.</p>
-    <p>Have an idea? Read our <a href="{{ site.repos[0].url }}/blob/master/CONTRIBUTING.md">contribution guidelines</a>.</p>
+  <div class="logo-links">
+    <a href="https://18f.gsa.gov/">
+      <img src="{{ site.baseurl }}/assets/img/logo-18f.png" alt="18F">
+    </a>
+    <a href="http://www.gsa.gov/">
+      <img src="{{ site.baseurl }}/assets/img/logo-gsa.png" alt="GSA">
+    </a>
+    <a href="https://www.usds.gov/">
+      <img class="logo-usds" src="{{ site.baseurl }}/assets/img/logo-usds.png" alt="USDS">
+    </a>
   </div>
-  <div class="usa-width-one-half usa-end-row">
-    <p>As a work of the United States government, this project is <a href="{{ site.repos[0].url }}/blob/master/LICENSE.md">in the public domain</a>.</p>
+  <div class="footer-text">
+    <div class="footer-text-one-half">
+      <p>Feedback? Create an issue on the <a href="{{ site.repos[0].url }}">GitHub repository</a> or email us at <a href="mailto:uswebdesignstandards@gsa.gov">uswebdesignstandards@gsa.gov</a>.</p>
```

Comment:
Since the link's text is "Github repository" I _think_ it should stay the repo URL or main issues page, but we could link to a new issue on "Create an issue" since it's an action phrase. What do you think @kategarklavs?